src/inputs/Table/components/FullscreenButton/FullscreenButton.js

```diff
@@ -25,4 +36,12 @@ FullscreenButton.propTypes = {
    * Function used to translate parent table in fullscreen
    */
   toggleFullscreen: PropTypes.func.isRequired,
+  /*
+   * Boolean to define is the button is disabled
```

Comment:
The description isn't clear, do you want to say _Boolean that defines whether button is disabled_ ?

Reply:
DONE